include/freetds/thread.h

```diff
@@ -412,6 +412,16 @@ static inline int tds_cond_timedwait(tds_condition *cond, tds_mutex *mtx, int ti
 }
 
 # endif
+#else
+static inline int tds_mutex_noop(void)
```

Comment:
Fair question.  I don't specifically care about any *platforms* that can't support threads, but a handful of the *configurations* we routinely test effectively use `--disable-threadsafe` and break without these changes.  ("Effectively" because we plug your sources and headers into our own build system.)